pandas/io/tests/parse_dates_tests.py

```diff
@@ -0,0 +1,471 @@
+"""
+Tests date parsing functionality for all of the
```

Comment:
idea is then you can easily strip out related things and add as separeat files in a nice way. (and don't crows the namespace)